### DISCUSSION TITLE
Release v1.0.14 / 5.55.14

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1720] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixed a security issue.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# ChangeLog

### Added
- added notification to warn that the default update checker is lagging behind the newest release on GitHub, to ensure that only bug-free builds are offered as updates [#1682](https://github.com/sandboxie-plus/Sandboxie/issues/1682)
- added main browsers to BlockSoftwareUpdaters template (by Dyras) [#1630](https://github.com/sandboxie-plus/Sandboxie/pull/1630)
- added a warning when Sandboxie-Plus.ini is not writeable [#1681](https://github.com/sandboxie-plus/Sandboxie/issues/1681)
- added clean-up for critical sections (by chunyou128) [#1686](https://github.com/sandboxie-plus/Sandboxie/pull/1686)

### Changed
- improved command line handling for breakout processes [#1655](https://github.com/sandboxie-plus/Sandboxie/issues/1655)
- disabled SBIE2193 notification (by isaak654) [#1690](https://github.com/sandboxie-plus/Sandboxie/pull/1690)
- improved error message 6004 [#1719](https://github.com/sandboxie-plus/Sandboxie/issues/1719)

### Fixed
- fixed dark mode issue with the new tray list
- fixed not showing a warning when Sandboxie-Plus.ini is not writeable [#1681](https://github.com/sandboxie-plus/Sandboxie/issues/1681)
- fixed issue with software compatibility checkbox (thanks MitchCapper) [#1678](https://github.com/sandboxie-plus/Sandboxie/issues/1678)
- fixed issue with events on box closure not always being executed [#1658](https://github.com/sandboxie-plus/Sandboxie/issues/1658)
- fixed memory leaks in key_merge.c
- fixed issue enumerating registry keys in privacy mode
- fixed settings issue introduced in 1.0.13 [#1684](https://github.com/sandboxie-plus/Sandboxie/issues/1684)
- fixed crash issue when parsing firewall port options
- FIXED SECURITY ISSUE: in certain cases a sandboxed process could obtain a handle on an unsandboxed thread with write privileges [#1714](https://github.com/sandboxie-plus/Sandboxie/issues/1714)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.0.14'>Release v1.0.14 / 5.55.14</a>.</em>

**Comment by BilleBarrett** [comment]
I'm not sure what is going on but I have had to remove version 14 from 2 laptops I installed it on yesterday
In both cases SBie.drv could not be removed and after I uninstall everything and deleted sanboxie.ini and all associated folders I reinstalled version 13 and everything worked OK
Let me know that I have the update on my desktop and I've had no problem
Please advise
[sandboxie.txt](https://github.com/sandboxie-plus/Sandboxie/files/8343059/sandboxie.txt)

**Reply by isaak654** [reply]
> 09:17:10.145   chrome.exe (15776): SBIE2310 Name buffer is approaching overflow (-1)   

I've received a similar overflow message that ends with a crash while browsing with a sandboxed session of Chrome 99.0.4844.82 64bit on Plus 1.0.14 x64, but it doesn't seem to happen with Chrome 99.0.4844.82 32bit. About the rest you mentioned, fortunately I have not encountered any further issues.

**Comment by BilleBarrett** [comment]
All my clients are always up to date with all their software, but it also happened to Microsoft Edge 
I’m running Chrome 99.0.4844.82, I’m not having the same problems
I only updated 2 clients so far

Thanks for the reply

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.***
Is your computer up to date and secure “NO” lets talk
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Reply by isaak654** [reply]
In case you have a crash dump to share, you need to zip the .dmp file locally and drag the resulting compressed file to a new GitHub message box (or an existing one). For further instructions: [working-with-advanced-formatting/attaching-files](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files)

**Comment by DavidXanatos** [comment]
hmm... its strange that this isues appeared, 1.0.14 is a small mostly incremental change over the 1.0.13 except when you enable privacy mode on a sandbox.
I'm looking into that issue

**Comment by DavidXanatos** [comment]
I have build 2 test sbiedll.dll's xanasoft.com/Downloads/test_1.0.14.zip please try out with which of it if any the chrom works again.
the strange thign is here all chromes work fine so its some strange fringe issue.

**Comment by BilleBarrett** [comment]
Where do I get them ?

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.***
Is your computer up to date and secure “NO” lets talk
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by BilleBarrett** [comment]
I just go a email from a client that his chrome was not working and that Sandboxie was posting that he had recovery to do but there was nothing

I uninstalled Sandboxie 14 and reinstalled Sandboxie 13 and everything was OK

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.***
Is your computer up to date and secure “NO” lets talk
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
